website/docs/internals/debugging.md

```diff
@@ -0,0 +1,29 @@
+import Tags from '@theme/Tabs';
```

Comment:
Currently, the **Internals** tab is meant to contain architectural information rather than end-user stuff. Fow now, can you please move this to the `docs/introduction/` folder and put this at the top of the file so it is at the bottom of the list:

```
---
sidebar_position: 60
---
```